docs/content/stable/admin/yb-admin.md

```diff
@@ -1372,6 +1428,34 @@ To display a list of tables and their UUID (`table_id`) values, open the **YB-Ma
 
 {{< /note >}}
 
+#### list_cdc_streams
```

Comment:
Looks like I picked up a CDC change with this one, too. That's good, albeit unexpected. :)